manifests/init.pp

```diff
@@ -72,6 +72,14 @@
 #                                  (30 or 30s), minutes (30m), hours (6h), days (2d),
 #                                  or years (5y).
 #
+# $autosign::                      Eanble or disable autosign or change location
+#                                  of autosign.conf or autosign script.  If this
+#                                  is set to a script, make sure that script
+#                                  considers the content of autosign.conf, as it
+#                                  might break Foreman functionality if it
+#                                  doesn't.
+#                                  type:string or boolean
```

Comment:
I think we can just keep it ```type:string``` since you can enter the string ```true``` or ```false``` and it will work.